.github/workflows/issues.yml

```diff
@@ -13,37 +13,46 @@ jobs:
   issues:
     runs-on: ubuntu-latest
     steps:
-      - name: Generate token for GitHub App
-        id: generate-token
+      - name: Generate token for GitHub App in the rstudio org
+        id: generate-token-rstudio
         uses: actions/create-github-app-token@v2
         with:
           app-id: 1064714
           private-key: ${{ secrets.POSIT_PLATFORM_PROJECTS_PEM }}
+          owner: rstudio
+      - name: Generate token for GitHub App in the posit-dev org
+        id: generate-token-posit-dev
+        uses: actions/create-github-app-token@v2
+        if:
```

Comment:
does empty `if:` work?

Reply:
lol, apparently so! I started to check to only do this if the label is for connect but the VS Code plugin for GHA didn't like the syntax, and I thought I deleted this. I'll clean it up.